orangecontrib/text/widgets/owconcordance.py

```diff
@@ -0,0 +1,265 @@
+from typing import Optional
+
+from AnyQt.QtCore import Qt, QAbstractTableModel, QSize, QItemSelectionModel, QItemSelection
+from AnyQt.QtWidgets import QSizePolicy, QApplication, QTableView, QStyledItemDelegate
```

Comment:
Hm, I thought I told PyCharm to break at 80, but it looks like it's using default 120. :/ It did seem excessively long, now when I think of it....